src/System.Management.Automation/namespaces/FileSystemProvider.cs

```diff
@@ -8278,7 +8280,7 @@ internal static bool IsReparsePointLikeSymlink(FileSystemInfo fileInfo)
                 fullPath = PathUtils.EnsureExtendedPrefix(fullPath);
             }
 
-            using (SafeFindHandle handle = FindFirstFileEx(fullPath, FINDEX_INFO_LEVELS.FindExInfoBasic, ref data, FINDEX_SEARCH_OPS.FindExSearchNameMatch, IntPtr.Zero, 0))
```

Comment:
Can the structs `FINDEX_INFO_LEVELS` and `FINDEX_SEARCH_OPS` be removed now?

Reply:
It's better to remove all if they are not in use anymore. If for some reason we need to revert this change, all will be reverted back anyways.

Reply:
I am exploring now how we can make our providers better in particular File Provider and may need it again. I don't mind removing it for now though.